pkg/storage/export/export/export.go

```diff
@@ -1409,3 +1410,12 @@ func (ctrl *VMExportController) createExportHttpDvFromPVC(namespace, name string
 	}
 	return nil
 }
+
+func (ctrl *VMExportController) getExportVolumeName(pvc *corev1.PersistentVolumeClaim, vmExport *exportv1.VirtualMachineExport) string {
```

Comment:
My goal is to avoid basically this if statement. If you look at https://github.com/kubevirt/kubevirt/blob/main/pkg/storage/export/export/export.go#L470-L478 you will see a few functions being passed in based on the type of source. The thought was to essentially have 2 functions, one for snapshot and one for the other two that look like this:
```go
func (ctrl *VMExportController) getExportVolumeNameSnapshot(pvc *corev1.PersistentVolumeClaim, vmExport *exportv1.VirtualMachineExport) string {
  return strings.TrimPrefix(pvc.Name, fmt.Sprintf("%s-", vmExport.Name))
}

func (ctrl *VMExportController) getExportVolume(pvc *corev1.PersistentVolumeClaim, vmExport *exportv1.VirtualMachineExport) string {
  return pvc.Name
}
```
And pass those functions to handleSource. This is slightly more flexible in how we handle the volume names. This is in case we have other types of sources that require more special handling.

Right now yours is simpler since we aren't passing the functions in. But if we get more sources that require different handling we end up with if else if type statements.

Reply:
Yeah, I get your point and I can change it if preferred, but it's true that passing that function as argument until reaching `getLinks` seemed too much when the same result could be achieved by having one slightly smarter general function. Other than that, just one question: Are you ok with keeping the same URLs (using the restore PVC name) and just changing the volume name as I currently do in the PR?

Reply:
I am a little confused and have to look more deeply but wouldn't changing the volumes passed to the export server but not the URLs basically give us the same issue we have now with the 404s?

Reply:
What I mean is to not modify the PVC name used to generate the URLs in the status (as we did before this PR), so we don't need to change the ENV variables that we pass to the export server. We would only modify it in the `name` section of the vmexport status, so users can recognize their original volume.

In the status of a hypothetical `example-export` vmexport we would see something like this:

```
     volumes:
      - formats:
        - format: raw
          url: https://virt-export-example-export.example.svc/volumes/example-export-example-disk/disk.img
        - format: gzip
          url: https://virt-export-example-export.example.svc/volumes/example-export-example-disk/disk.img.gz
        name: example-disk
```
This is simpler and would only require using the new `getExportVolumeName` function in `getLinks`.

Reply:
okay gotcha, yeah that works for me.